backend/frontend/templates/frontend/user/board/js_resize_scroll_page.html

```diff
@@ -0,0 +1,12 @@
+<script>
+  window.addEventListener("resize", function () {
+    const columns = document.querySelectorAll(".grid-cols-5 > div");
+    const windowHeight = window.innerHeight;
+
+    columns.forEach(function (column) {
+      column.style.height = windowHeight - 160 + "px";
+    });
+  });
+
+  window.dispatchEvent(new Event("resize"));
+</script>
```

Comment:
forgot to mention that the function and event handling should be moved to the static folder, and maybe even one of the existing files

Reply:
> forgot to mention that the function and event handling should be moved to the static folder, and maybe even one of the existing files

existing files you mean the page.html  template ?